src/renderer/webgl/shaders/RoundedRectangleShader.mjs

```diff
@@ -156,7 +226,8 @@ RoundedRectangleShader.fragmentShaderSource = `
         }
 
         float b = boxDist(vTextureCoord.xy * resolution - halfRes, halfRes - 0.005, r);
-        vec4 tex = mix(vec4(0.0), color * fillColor, fillMask(b));
-        gl_FragColor = mix(tex, vColor * strokeColor, innerBorderMask(b, stroke));
+        vec4 color = mix(texture2D(uSampler, vTextureCoord) * vColor, fillColor * alpha, fill);
```

Comment:
I reverted some of it back, but i think the strokeColor wasn't functional if the texture color/vColor unless it was white so that's why i started comparing it to an alpha color. The background/texture itself should blend like it did before. I think we need to have a convo about this soon.